docs/guides/mapeo-http-server.md

```diff
@@ -1,28 +1,37 @@
-# Mapeo's Media Server
+# Mapeo's HTTP Server
 
-Each Mapeo manager instance includes an embedded HTTP server that is responsible for serving media assets over HTTP. Each server is responsible for handling requests for assets that can live in any Mapeo project (the URL structure reflects this, as we will show later on).
+Each Mapeo manager instance requires an adjacent Fastify server instance that is responsible for serving assets over HTTP. Each Fastify instance is responsible for handling requests for assets that can live in any Mapeo project (the URL structure reflects this, as we will show later on).
 
 Some boilerplate for getting started with a Mapeo project:
 
 ```js
+// Create Fastify instance
+const fastify = Fastify()
+
+// Create FastifyController instance for managing the starting and stopping the Fastify server (handles it more gracefully and allows pausing and restarting)
+const fastifyController = new FastifyController({ fastify })
```

Comment:
I think maybe add something to this about this being optional - on Desktop we would only need to call `fastify.listen()`, the use of FastifyController is only necessary for mobile where we need to start and stop the server (because Android doesn't like us running a server while the app is in the background)

Reply:
addressed via https://github.com/digidem/mapeo-core-next/pull/440/commits/3fb05ddf439efde39531c72407e0ea6c523a3af8